identifiers/country-eg.csv

```diff
@@ -5,172 +5,172 @@ ocd-division/country:eg/ed:eastern_delta,Eastern Delta
 ocd-division/country:eg/ed:northern_central_southern_upper_egypt,"Northern, Central and Southern Upper Egypt"
 ocd-division/country:eg/ed:western_delta,Western Delta
 ocd-division/country:eg/governorate:alx,Al Iskandarīyah
-ocd-division/country:eg/governorate:alx/ed:1,Al Iskandarīyah - 1st district
-ocd-division/country:eg/governorate:alx/ed:2,Al Iskandarīyah - 2nd district
-ocd-division/country:eg/governorate:alx/ed:3,Al Iskandarīyah - 3th district
-ocd-division/country:eg/governorate:alx/ed:4,Al Iskandarīyah - 4th district
-ocd-division/country:eg/governorate:alx/ed:5,Al Iskandarīyah - 5th district
-ocd-division/country:eg/governorate:alx/ed:6,Al Iskandarīyah - 6th district
+ocd-division/country:eg/governorate:alx/ed:1,Al Iskandarīyah - 1 district

```

Comment:
I think these should be "District 1" or "1st district" but not "1 district" as that is not a common presentation.

Reply:
I updated to 1st district